errands/widgets/task_list.py

```diff
@@ -63,22 +76,45 @@ def build_ui(self):
             sensitive=False,
         )
         self.scroll_up_btn.connect("clicked", lambda *_: scroll(self.scrl, False))
+        self.left_toggle_sidebar_bin = Adw.Bin(
+            child=self.left_toggle_sidebar_btn
+        )
+        self.right_toggle_sidebar_bin = Adw.Bin(
+            child=self.right_toggle_sidebar_btn
+        )
 
         # Header Bar
         hb = Adw.HeaderBar(title_widget=self.title)
-        hb.pack_start(self.toggle_sidebar_btn)
+        hb.pack_start(self.left_toggle_sidebar_bin)
         hb.pack_start(self.delete_completed_btn)
+        hb.pack_end(self.right_toggle_sidebar_bin)
         hb.pack_end(self.scroll_up_btn)
 
         # ---------- BOTTOMBAR ---------- #
 
-        toggle_sidebar_btn = Gtk.ToggleButton(
+        left_toggle_sidebar_btn = Gtk.ToggleButton(
             icon_name="sidebar-show-symbolic",
             tooltip_text=_("Toggle Sidebar"),
         )
-        toggle_sidebar_btn.bind_property(
+        right_toggle_sidebar_btn = Gtk.ToggleButton(
+            icon_name="sidebar-show-right-symbolic",
+            tooltip_text=_("Toggle Sidebar"),
+        )
+        left_toggle_sidebar_bin = Adw.Bin(
+            child=left_toggle_sidebar_btn
+        )
+        right_toggle_sidebar_bin = Adw.Bin(
+            child=right_toggle_sidebar_btn
+        )
```

Comment:
Here too

Reply:
Here it's actually useless as the whole bottom toolbar is hidden or revealed when it should be visible, so the left/right sidebar preference is the only factor affecting the visibility of the button itself. I'll improve that!